src/views/v2/components/loader/loader.php

```diff
@@ -14,14 +14,18 @@
  * @version 5.0.4
  */
 
-$classes = $this->get( 'classes' ) ?: [];
+$classes        = $this->get( 'classes' ) ?: [];
+$visible        = tribe_is_truthy( $this->get( 'visible' ) ) || false;
```

Comment:
```suggestion
$visible        = tribe_is_truthy( $this->get( 'visible' ) );
```
This method already returns a bool, I think there's no need to have the OR false.